src/main/java/io/split/android/client/SplitClient.java

```diff
@@ -62,4 +62,10 @@
      * Flushes all memory allocated queues which should be stored on disk or sent to server.
      */
     void flush();
+
+    /**
+     * Checks if cached data is ready to perform treatment evaluations
+     * @return true if getTreatment is ready to call
```

Comment:
I would say  "true' if the sdk is ready, if false, calls to getTreatment will return control